src/modules/common/store.ts

```diff
@@ -80,8 +80,7 @@ const { storageMiddleware, loadStorageMiddleware } = createStorageMiddleware({
     ['scene', 'present'],
     ['ui', 'dashboard'],
     ['auth', 'data'],
-    ['sync', 'localProjectIds'],
-    ['identity', 'data']
```

Comment:
By removing this, the identity is not persisted in local storage anymore. It is now persisted in the iframe so there is no reason for it.